specs/src/networking.md

```diff
@@ -54,7 +57,10 @@ When receiving a new block proposal `proposal` from the network, the following s
 Defined as `MsgWirePayForData`:
 
 ```protobuf
-{{#include ./proto/wire.proto:MsgWirePayForData}}
+// MsgWirePayForData message definition (removed from specs)
```

Comment:
MsgWirePayForData it not a real type so the protobuf and entire section should be removed entirely.